src/site/_includes/macros/tag-pills.njk

```diff
@@ -0,0 +1,14 @@
+{#
+  @param {string[]} tags - An array of tag keys to display.
+  @param {string} [className] - A space separated list of classes to apply.
+#}
+{% macro tagPills(tags, className = '') %}
+<div class="card__tags cluster gap-top-base {{className}}" aria-label="tags for this post">
```

Comment:
This class name "card__tags suggest this macro is used only in context of a post card. Is there another context where this will be used? If not, it is better to either leave it inside the post-card.njk, so that all card markup is in one place, or extract only pills (without a wrapper div) so that the tag pills can be reused e.g. in the post page, below a title or so. I think the previous version of this template, tag-chips.njk, was written with 2nd option in mind.

Reply:
Presently it's used only by `src/site/_includes/macros/post-card.njk`, but my thought is that it will also be used by `src/site/_includes/partials/post.njk`.

On the old design system both `src/site/_includes/macros/post-card.njk`, `src/site/_includes/partials/post.njk` used `src/site/_includes/macros/tag-chips.njk`. My thought was `src/site/_includes/macros/tag-pills.njk` would replace `src/site/_includes/macros/tag-chips.njk`.

Anyway, I'll remove the div wrapper and I think we should be good.